py-scripts/lf_ftp.py

```diff
@@ -260,11 +399,16 @@ def query_realclients(self):
                     self.user_list.append(device)
         logger.info("AVAILABLE DEVICES TO RUN TEST: %s", self.user_list)
         logging.info(self.user_list)
+        # Case 4: Config is False, no device list is provided, and no group is selected
+        if (self.config == False and len(self.device_list) == 0 and self.group_name is None):
+            logger.info("AVAILABLE DEVICES TO RUN TEST : {}".format(self.user_list))
```

Comment:
Please use the more modern Python format strings in the future (e.g. `print(f"devices: {self.user_list}")`). Fine as is.